### PR TITLE
styling

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

File: src/chartkit/components/HorizontalBar.js [hunk 1]
```diff
@@ -52,7 +55,8 @@ class HorizontalBar extends Component {
   }
 
   onMouseEnter(data, e) {
-    e.target.style.cursor = 'pointer';
+    //@NOTE Re-enable once graph is clickable
```
Comment (ciaranschutte) [inline src/chartkit/components/HorizontalBar.js]: V important. User Dashboard has charts with pointer for clickable currently. This breaks it.

Comment (nyanofthemoon) [reply src/chartkit/components/HorizontalBar.js]: @ciaranschutte Was requested to remove this for now by @rosibaj

Comment (nyanofthemoon) [reply src/chartkit/components/HorizontalBar.js]: @ciaranschutte No "easy" way to remove temporarily, so putting back.

Comment (rosibaj) [reply src/chartkit/components/HorizontalBar.js]: @ciaranschutte @nyanofthemoon Not sure on code here, but its _very_ important to have the pointer where needed, but not where not needed.  Can we work out a way for that to happen?

**Pull request merged**